packages/core/admin/admin/src/utils/createRoute.tsx

```diff
@@ -6,7 +6,9 @@ import { LoadingIndicatorPage } from '@strapi/helper-plugin';
 import { Route } from 'react-router-dom';
 
 interface LazyCompoProps {
-  loadComponent: () => Promise<{ default?: React.ComponentType } | React.ComponentType>;
+  loadComponent: () =>
+    | Promise<{ default?: React.ComponentType } | React.ComponentType>
+    | { default?: React.ComponentType | React.ComponentType };
```

Comment:
Is the second part correct? it's saying default is either undefined or React.ComponentType or React.ComponentType?

Reply:
You are right - I've fixed it.